arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts

```diff
@@ -639,17 +645,17 @@
 
 			core0run@40 {
 				//P0_VDD_CORE_0_RUN
-				compatible = "isil,isl68137";
+				compatible = "raa229639";
 				reg = <0x40>;
 			};
 			core1run@41 {
 				//P0_VDD_CORE_1_RUN
-				compatible = "isil,isl68137";
+				compatible = "raa229639";
```

Comment:
This change is unrelated to the commit message. Please update either the commit message or split the patch.